docs/reference/migration/migrate_7_10.asciidoc

```diff
@@ -18,54 +18,182 @@ See also <<release-highlights>> and <<es-release-notes>>.
 //tag::notable-breaking-changes[]
 
 [discrete]
-[[breaking_710_search_changes]]
-=== Search Changes
+[[breaking_710_security_changes]]
+=== Authentication changes
 
-[discrete]
-==== Missing checks for `index.max_docvalue_fields_search`
-The setting `index.max_docvalue_fields_search` limits the number of fields that
-can be retrieved through `docvalue_fields` in a search. Previously we only
-checked this limit when `docvalue_fields` appeared in the top-level search
-request body. Now we also enforce the limit when loading doc value fields in
-an `inner_hits` section or in a `top_hits` aggregation.
+[[api-keys-require-name-property]]
+.API keys now require a `name` property.
+[%collapsible]
+====
+*Details* +
+The `name` property is now required to create or grant an API key.
+
+[source,js]
+----
+{
+    "...": "...",
+    "api_key": {
+      "name": "key-1"
+    }
+}
+----
+// NOTCONSOLE
+
+*Impact* +
+To avoid errors, specify the `name` property when creating or granting API keys.
+====
 
 [discrete]
-[[breaking_710_networking_changes]]
-=== Networking Changes
+[[breaking_710_indices_changes]]
+==== Indices changes
```

Comment:
I think this should be a peer section to the Authentication section, not a child.
```suggestion
=== Indices changes
```